include/core/parameters.h

```diff
@@ -115,10 +115,13 @@ namespace Parameters
     // Max CFL
     double maxCFL;
 
+    // Max dt
```

Comment:
Just to be consistent with the comment in `simulation_control.h`
```suggestion
    // Max time step
```